File: performanceOPT/dual_process/src/main/java/com/test/dual_process/MainActivity.java
package com.test.dual_process;

import android.content.Intent;
import android.os.Bundle;

import com.test.dual_process.service.LocalService;
import com.test.dual_process.service.MyJobService;
import com.test.dual_process.service.RemoteService;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        startService(new Intent(this, LocalService.class));
        startService(new Intent(this, RemoteService.class));
        MyJobService.startJob(this);
    }
}

File: cnw_jetpack_project/2-7-liveData/app/src/main/java/com/cniao5/demo/MainActivity.kt
package com.cniao5.demo

import android.graphics.Color
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)
		setContentView(R.layout.activity_main)


		ph_main.setContentId(R.id.tv_holder)

//		group.visibility = View.GONE
	}
}